src/js/providers/html5.js

```diff
@@ -140,10 +140,10 @@ define([
             _beforecompleted = false,
 
             _fullscreenState = false,
-
-            // Video Text Tracks
+            // Video Tracks
             _textTracks,
-            _currentTextTrackIndex = -1;
+            _currentTextTrackIndex = -1,
+            _currentAudioTrackIndex = -1;
```

Comment:
Only when there are tracks in the new source. You should clear these on init/load.